### PR TITLE
Move start button to first page of wizard

### LINKED ISSUE #4413: Wizard - move start button to first page
### Description

Desired UX:

First page:
- move the button to start the migration to the first page
- Cancel should close dialogue

Second page:
- Finish button should be disable until the migration has finished
- window close/cancel should be disabled until migration has finished, then Cancel should close the dialogue.

We'll look at making the migration cancellable separately

### PR DESCRIPTION
**Pull request by duncanp-sonar**
Fixes #4413

### CODE REVIEW

**Comment by duncanp-sonar** [comment]
The pages now look like this:

image

image

File: src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml [hunk 1]
```diff
@@ -77,16 +79,14 @@
...
-            </Grid>
+            <DockPanel DockPanel.Dock="Bottom">
+                <Button HorizontalAlignment="Right" Content="Finish" x:Name="finishButton" IsEnabled="false" IsCancel="True" Style="{DynamicResource Button_Style}"/>
+            </DockPanel>
 
             <!-- Last child - fills the body of the panel -->
             <DockPanel LastChildFill="True">
                 <TextBlock DockPanel.Dock="Top">Progress:</TextBlock>
-                <ListBox x:Name="progressList" Margin="0,8,0,10" BorderBrush="LightGray" ScrollViewer.VerticalScrollBarVisibility="Auto" />
+                <ListBox x:Name="progressList" Margin="0,8,0,10" BorderBrush="LightGray" ScrollViewer.VerticalScrollBarVisibility="Auto" Height="350" />
```
Comment (duncanp-sonar) [inline src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml:89]: Fixed height to stop it resizing the dialogue. A vertical scrollbar will appear if needed.

**Review by bigfluffycookie — COMMENTED** [review]
Looks good just one question

File: src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml [hunk 2]
```diff
@@ -21,17 +21,19 @@
     </Window.Resources>
 
     <Grid TextBlock.FontSize="15">
+        <Separator VerticalAlignment="Top" />
         <DockPanel x:Name="StartWindow" Margin="10" LastChildFill="True" Visibility="Visible" >
             <DockPanel DockPanel.Dock="Bottom">
-                <Button DockPanel.Dock="Left" Content="Cancel" IsCancel="True" HorizontalAlignment="Left"  Style="{DynamicResource Button_Style}" />
-                <Button DockPanel.Dock="Right" Content="Continue" Click="NavigateToMigrationProgressPage" HorizontalAlignment="Right" Style="{DynamicResource Button_Style}"/>
+                <Button x:Name="btnPage1_Cancel" DockPanel.Dock="Left" Content="Cancel" IsCancel="True" IsDefault="true" HorizontalAlignment="Left"  Style="{DynamicResource Button_Style}" />
```
[the remark this answers is not in the document]
Comment (duncanp-sonar) [reply src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml:27]: Yes, I think that would be the normal behaviour. My thinking was that it means the user can't accidentally start the migration - they would have to actually click on Start, or tab the focus to the Start button then press Enter. But as you say, it's not the normal convention. I'll change it, and we can run it past Marco later.

[the remark this answers is not in the document]
Comment (bigfluffycookie) [reply src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml:27]: In that case I'd just leave out the default entirely (or just add it to the finish button)

[the remark this answers is not in the document]
Comment (duncanp-sonar) [reply src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml]: I'll leave Start as the default for now. We can tweak it later if needed.

**Pull request merged**